catalog.ts

```diff
@@ -182,6 +182,20 @@ export class Catalog {
     }
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public async searchDocuments(query: string): Promise<any> {
```

Comment:
If we do decide to add this to the SDK we need to add strongly typed output if possible. In general we should never use `any` unless we have a very good reason.